### DISCUSSION TITLE
lazy lazer design rating post (x-post) - UI/Interface

### DISCUSSION THREAD

**Discussion by finesseandstyle** [ppy/osu#30436] [Ideas]
I'm not the OP from this post, I'm just posting this from /r/osugame so it's more visible to developers because I do agree with the criticisms:

yup youve probably seen me around shilling for lazer and i do think its great but i am getting very angry at the design process for the client right now and i thought id share my gripes with both the song select and results screen (both current and proposed designs) i am typing this without checking for spelling errors or revising any text btw just so you know

————————————————————————————————————

ok so for the song select lets run through this on default settings since it's what everyone sees, for some reason they decided that the background both isnt important enough to be unblurred and that it should be almost completely covered with default scaling but also important enough to take up 40% of the leaderboard real estate and get copypasted 3 times like bro wtf is this

![rztkmuyw08xd1 1](https://github.com/user-attachments/assets/eb1e7bc1-5522-461b-94d4-fb9f29c32358)

as well the leaderboard panels are way too big clocking in at 80px tall and 726px wide vs stable's 68px tall and 526px wide (80% and 70% size respectively) but the carousel is fine size wise if not stretching a bit too far inwards (covering the bg yet again). almost all of the scale related complaints are solved at 80% scale so I wish that was the default, and the wedge needs to screw off to a roblox clone of osu. OVERALL RATING: 3/10 standard 5/10 at 80% zoom

————————————————————————————————————

for the most recent song select redesign i could find on figma its a bit better in most aspects, some general impressions have been annotated in the image, for example the bg doesnt seem blurred by default anymore which is cool and the wedge is improved a bit but its still not great (nobody cares about fail rate) and the entirety of song select seems too heavy on the left, since flyte seems obsessed with slanting as many things as he can but you cant really slant the carousel

![k2svklq238xd1 1](https://github.com/user-attachments/assets/3f030a3a-4bdb-480c-892a-28536006e111)

I think killing the wedge is a lost cause flyte seems obsessed with it, like look at this old rendition ![old rendition](https://github.com/user-attachments/assets/f1ae3316-159a-4d4b-a5dd-a13715f16c40)

(banished to discord cdn because its a waste of post real estate). at least you cant really see the third copy pasted background since its covered up with actual useful information. (lol at the flat osu button btw peppy shut that one down asap) overall id be kinda happy with this maybe its a 6/10 not sure about with scaling options since its not in game

————————————————————————————————————

ok now its results screen time its time for some brutal murder. there are 2 proposals i am going to include and i want you to guess which one i prefer from the feedback i give. so for the current one im just gonna rate it extended because im not playing on a portrait mode phone, it looks pretty meh because you literally cant see the BG (try guessing which map this is if you cant read kana LOL) and the useful info is about 1/20th of the screen real estate

![5zwwj2lq48xd1 1](https://github.com/user-attachments/assets/910a4dd9-b615-427d-9877-72c125a94033)

i also dont like scrolling but the scrollage doesnt exist on 80% zoom since all the extended stats fit, and peppy literally made a github issue saying "results screen sucks" let alone the fact that you're permanently at the end of a long list of players better than you which hurts my ego 1/10

————————————————————————————————————

for the first redesign, lets look at the design teased in a lazer updates video i wont bother finding again, so basically he decided to make the wheel barely anyone cares about 10x bigger and copy paste the background in it for some reason (just show the ACTUAL background please) and the ratio of useful to useless information actually gets worse somehow, as well as the judgement numbers being smaller than in the old version somehow? it feels like he decided that fixing the results screen meant moving the score card from the middle to the right and keeping almost everything else the same

![p6g3vz8n58xd1 1](https://github.com/user-attachments/assets/eabce1eb-f2b9-4a45-95c1-b295e68cf01e)

now theres a load more button and i really do not like it, it seems like you wont be able to get the UR and the judgements in the same screenshot and they removed all the dead space by squishing everything to the top proving its just there because they really dont want you to see the background for some reason, and i dont want to have to scroll to see everything on the very right when you are wasting all the space on the left (ignore radiant rank and stuff theyre just flyteisms). not to mention there are like 5 different fonts in the extended statistics for some reason but thats just polish and i've been ignoring that so far and just focusing on the layouts

![1g4gb7xf68xd1 1](https://github.com/user-attachments/assets/d6872c36-5beb-442f-9c84-ab11e07b3fd9)

this whole thing feels like a downgrade from the current results screen somehow? 0.5/10 for effort

————————————————————————————————————

ok so for the final design review lets go over something made by a volunteer, as i said a second ago i am indeed ignoring polish so just bear with me when i only review the layout. it's just a mishmash of other flyte design elements and i think it fits way better with the proposed song select layout above unlike that other results screen proposal, the judgement counts are a decent size if not a bit small still and the extended stats look integrated into the design instead of an afterthought

![6uxy47zq78xd1 1](https://github.com/user-attachments/assets/ee196222-c9cd-4d5a-929d-3a4aaebe0bef)

the useful portion is now sitting about 40% of the screen instead of 5% which is a major upgrade, the placement of everything is a bit weird with the judgements just floating in the middle of the BG instead of being attached to the side or something, but this whole design feels more player focused. genius integration of the new song select button style at the bottom btw i forgot to annotate that. i just realized where tf is the back button on the other design LOL anyway this one is a 7/10 or maybe even a 7.5/10 this one is pretty nice.

————————————————————————————————————

for an afterword, to any lazer devs out there please take this as a way to say that lazer is awesome but the designer needs to be in touch with the community and u should be iterating more, ive had to stare at the old song select for 3 years now and it gets annoying after a while watching the song select PRs get bikeshedded to hell

**Comment by mixalisnew** [comment]
copy pasta is coming to github

**Comment by bytenekos** [comment]
I really like the last design for the results screen. I'm imagining the song "banner" on the top left "dynamically" resizing when moving from results screen to song select and I think that'd be sick

**Comment by peppy** [comment]
Thanks but most of this has been addressed countless times. Half waiting on design half waiting on implementation.
